airbrakes/imu/imu_data_packet.py

```diff
@@ -0,0 +1,152 @@
+"""Module for describing the datapackets from the IMU"""
+
+
+def _mro_slots(obj) -> list[str]:
```

Comment:
can you explain this method better? I'm a little lost

Reply:
Basically when you access `.__slots__` of an instance, it only gives you the slots defined for that class. But we also want the slots of its parent class, i.e. IMUDataPacket for the `timestamp`. MRO = method resolution order - https://docs.python.org/3/library/stdtypes.html#class.__mro__

Reply:
oh cool